src/main/java/mate/academy/rickandmorty/mapper/CharacterMapper.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.rickandmorty.mapper;
+
+import mate.academy.rickandmorty.dto.external.ApiCharacterDto;
+import mate.academy.rickandmorty.dto.internal.CharacterResponseDto;
+import mate.academy.rickandmorty.model.Character;
+import mate.academy.rickandmorty.model.Gender;
+import mate.academy.rickandmorty.model.Status;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CharacterMapper {
+    public CharacterResponseDto toResponseDto(Character character) {

```

Comment:
be consistant with naming. It's either toCharacterResponseDto() and to Character() or toDto() and toModel()
```suggestion
    public CharacterResponseDto toDto(Character character) {
```